packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_ExecutionManagerWrapper.sol

```diff
@@ -118,6 +118,25 @@ library Lib_ExecutionManagerWrapper {
         return abi.decode(returndata, (address));
     }
 
+    /**
+     * Calls the ovmCHAINID opcode.
+     * @return Chain ID of the current network.
+     */
+    function ovmCHAINID()
+        internal
+        returns (
+            uint256
```

Comment:
@smartcontracts - Did you mention something about using a smaller `uint` for the chainID? Or was that with regard to the nonce that Proto and others have brought up?

I'm not familiar with the DTL RLP codepath but maybe submitting a crazy high `v` would cause unexpected behavior / cause us to decode a transaction incorrectly. Haven't confirmed this but was wondering if it was on your radar

Reply:
Yes, I was planning to make that change in another PR following this one. We'll match the limits up with whatever geth is using. Just didn't want to add that change here b/c too much for a single PR.